blocks/form/form.js

```diff
@@ -438,8 +438,14 @@ export default async function decorate(block) {
     let formURL = formLink.href;
     const config = readBlockConfig(block);
     if (formURL.endsWith('contact.json')) {
-      const locale = getMetadata('locale') || 'en';
-      formURL += (locale !== 'en' ? `?sheet=${locale}` : '');
+      const localeStr = getMetadata('locale') || 'en';
+      const locale = localeStr.split('-')[0];
+      const region = localeStr.split('-')[1];
+      if (locale === 'en' && region && region.toLowerCase() === 'gb') {
```

Comment:
for GB we are using english language sheet?

Reply:
No uk language sheet